docs/source/tutorial/upstream_collection.rst

```diff
@@ -973,6 +973,69 @@ wavlablm_ms_40k
 - Trained from scratch and use a multilingual k-means from the training data with a multi-stage training
 
 
+Multiresolution HuBERT (MR-HuBERT)
+----------------------
+`Multi-resolution HuBERT: Multi-resolution Speech Self-Supervised Learning with Masked Unit Prediction <https://openreview.net/pdf?id=kUuKFW7DIF>`_
+
+.. code-block:: bash
+
+    @inproceedings{anonymous2023multiresolution,
+        title={Multi-resolution Hu{BERT}: Multi-resolution Speech Self-Supervised Learning with Masked Unit Prediction},
+        author={Anonymous},
+        booktitle={Submitted to The Twelfth International Conference on Learning Representations},
+        year={2023},
+        url={https://openreview.net/forum?id=kUuKFW7DIF},
+        note={under review}
+    }
+
+
+multires_hubert_custom
+~~~~~~~~~~~~~~~~~~~~~
+
+This entry expects you to provide the source of the checkpoint: :code:`ckpt`, which should be
+the local path or a url of the checkpoint converted by :code:`s3prl/upstream/multires_hubert/convert.py` (
+from a regular fairseq checkpoint.)
+For more available checkpoints, please check `https://github.com/facebookresearch/fairseq/blob/main/examples/mr_hubert/README.md`_
+Related converted checkpoints are also at `https://huggingface.co/s3prl/mr_hubert`_
+
+
+multires_hubert_base
+~~~~~~~~~~~~~~~~~~~~~
+
+- Unlabled Speech: LibriSpeech 960hr
+- K-means extracted from `hubert_base`_
+
+
+multires_hubert_large
+~~~~~~~~~~~~~~~~~~~~~
+
+- Unlabeled Speech: LibriLight 60khr
+- K-means extracted from `hubert_base`_
+
+
+multires_hubert_multilingual_base
+---------------------------------
+
+- Unlabeled Speech: Voxpopuli 100khr
+- K-means extracted from `hubert_base`_
+
+
+multires_hubert_multilingual_large400k
+--------------------------------------
+
+- Unlabeled Speech: Voxpopuli 100khr
+- K-means extracted from `hubert_base`_
+- Training steps 400k
+
+
+multires_hubert_multilingual_large600k
+--------------------------------------
+
+- Unlabeled Speech: Voxpopuli 100khr
+- K-means extracted from `hubert_base`_
+- Training steps 600k
+
+
```

Comment:
For the K-means extraction source, the current document implies that all the models are using the K-means from the 2nd iteration of HuBERT Base, could you help double check if this is true?
For example, this means `multires_hubert_base` is the 3rd iteration of the HuBERT training, which will not be comparable to `hubert_base`.

Reply:
Yeah, the current note is true. All the provided models shall be considered as the 3rd iteration of the HuBERT-training (not directly comparable to hubert_base). In our provided paper, we also conducted the experiments with the "comparable" hubert using the same 3rd iteration k-means.

In that case, do you suggest me submit those models (3rd iteration hubert) to the s3prl as well?

Reply:
Oh I think adding the 3rd iteration of HuBERT is not necessary. I was just curious and making sure that the note is correct. I have no further issue then. (You can still submit the 3rd iteration HuBERT if you want)